sdks/python/apache_beam/io/filesystem.py

```diff
@@ -302,23 +302,28 @@ def seek(self, offset, whence=os.SEEK_SET):
     """Set the file's current offset.
 
     Seeking behavior:
-      * seeking from the end (SEEK_END) the whole file is decompressed once to
-        determine it's size. Therefore it is preferred to use
-        SEEK_SET or SEEK_CUR to avoid the processing overhead
-      * seeking backwards from the current position rewinds the file to 0
+
```

Comment:
Unfortunately, this empty line is used as part of the syntax to indicate a new block. Removing this empty line causes the html generated to be incorrect.